packages/lib-polkadot/polkadot-signer.ts

```diff
@@ -0,0 +1,15 @@
+import { Signer, SignerResult } from '@polkadot/api/types'
+import { SignerPayloadRaw } from '@polkadot/types/types/extrinsic'
+import { DfnsWallet } from './index'
+
+export class PolkadotSigner implements Signer {
+    // Id we increment for each signature
+    private id: number
```

Comment:
Just curious, what is it for?

Reply:
The type `SignerResult` ask us to return a number. It feels safe to return a different one for each request. 

We can see an example here: https://github.com/polkadot-js/apps/blob/master/packages/react-signer/src/signers/AccountSigner.ts#L29C29-L29C30

I am afraid I don't know more about the underlying logic